### PR TITLE
Added Table of Contents and FAQs to the README

### PR DESCRIPTION
**Pull request by adarsh-jha-dev**
# Description

This pull request adds a Table of Contents and FAQs section to the README of the project. The Table of Contents helps users navigate the documentation more easily, and the FAQs section provides answers to common questions about Featureform.

## Type of change
Documentation Update

### Does this correspond to an open issue?

No

### Select type(s) of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

# Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have fixed any merge conflicts

### CODE REVIEW

**Comment by adarsh-jha-dev** [comment]
can any of the maintainers please check this PR and merge it if found meaningful?

File: README.md
Comment (ahmadnazeri) [inline README.md:32]: These links do not work with the exception of the last 4 links

**Review by ahmadnazeri — CHANGES_REQUESTED** [review]

File: README.md
Comment (adarsh-jha-dev) [reply README.md:32]: I have rectified this . Now those links are working

**Comment by adarsh-jha-dev** [comment]
Hey @ahmadnazeri , I have done the requested changes, could you please review them?

File: README.md
Comment (ahmadnazeri) [inline README.md:248]: Can you please remove the FAQs?

**Review by ahmadnazeri — CHANGES_REQUESTED** [review]
Thanks for fixing the links! Can you please remove the FAQs section then we should be good?

File: README.md
Comment (adarsh-jha-dev) [reply README.md]: Done

**Comment by adarsh-jha-dev** [comment]
@ahmadnazeri , I have done all the changes, I request you to please proceed further

**Comment by adarsh-jha-dev** [comment]
Hey @ahmadnazeri could you please let me know if this can be merged?

**Review by ahmadnazeri — APPROVED** [review]

**Comment by ahmadnazeri** [comment]
Thanks @adarsh-jha-dev, just approved it and will have to run the test pipeline before I can merge it

**Comment by adarsh-jha-dev** [comment]
> Thanks @adarsh-jha-dev, just approved it and will have to run the test pipeline before I can merge it

Sure, thank you

**Pull request closed without merging**
